package.json

```diff
@@ -95,6 +95,7 @@
   },
   "dependencies": {
     "babel-runtime": "6.9.2",
-    "vue": "1.0.24"
+    "vue": "1.0.24",
+    "vue-multiselect": "^1.0.1"
```

Comment:
I made a fiddle: http://www.webpackbin.com/4yy1cQ2OW
Just load the lib with a `script` tag. In the fieldMultiselect.vue check the lib loaded and register as a global component.